assets/ajax-cart.js.liquid

```diff
@@ -375,6 +408,7 @@ var ajaxCart = (function(module, $) {
 
       // If it has a data-line, update the cart
       if (line) {
+        $body.trigger('adjustCart.ajaxCart', [this, line, qty]);
```

Comment:
Another reason is access to the element which isn't available inside updateQuantity